rts/lra-examples/lra-jwt/wildfly/README.md

```diff
@@ -0,0 +1,75 @@
+# Securing LRA coordinator endpoints over Wildfly server using JWT
+
+ This quickstart will demonstrate how to secure LRA-coordinator endpoints using JSON Web Tokens(JWTs) when the lra-coordinator is deployed on Wildfly server
+
+ The `lra-coordinator WAR` expects to be the same as one at https://github.com/jbosstm/narayana/blob/master/rts/lra/coordinator-war/pom.xml. But the purpose is to compile into this one with a different web.xml, which would be used to define the Authentication/Authorization mechanism.
+
+
+## Follow along below steps:
+1. Setup web.xml before deploying the **lra-coordinator** war file. Here we define the secutiry mechanism for application endpoints i.e. JWT and also define the **Roles** and **endpoints** mapping for authorization.
```

Comment:
Should not be the deployment of the lra-coordinator defined as a separate step?

Reply:
I mean - how to deploy the `lra-coordinator`? I think it's not clear what that means - is it copying, is it running a maven command?

Reply:
User will have to place the `lra-coordinator` war file under `$JBOSS_HOME/standalone/deployments`

Reply:
And could that be mentioned in the `README` text, please?

Reply:
(if it's not part of the text already and I missed that)

Reply:
It is mentioned [here](https://github.com/jbosstm/quickstart/pull/300/files#diff-ef5e4bc473765ab97f19f64ee1a96fac71d95331c6e3401f2d3c210f35a17232R41)

Reply:
Ok, sorry, I missed that.